pkgs/games/katago/default.nix

```
@@ -29,14 +29,14 @@ assert !enableGPU -> (
 # of gcc.  If you need to use cuda10, please override stdenv with gcc8Stdenv
 stdenv.mkDerivation rec {
   pname = "katago";
-  version = "1.10.0";
+  version = "1.11.0";
   githash = "ff49d04ad6bcfa056c63492439a41e2f3bce0847";
```

```suggestion
  githash = "d8d0cd76cf73df08af3d7061a639488ae9494419";
```

Need to update the git hash for contrib.